macros/SpecName.ejs

```diff
@@ -848,24 +836,24 @@ var specList = {
         url  : 'http://dev.w3.org/2006/webapi/selectors-api2/'
     },
     'Shadow DOM':{
-        name : 'Shadow DOM',
-        url  : 'https://www.w3.org/TR/shadow-dom/'
+        name : 'DOM (Shadow trees)',
```

Comment:
I got more "sorry!" messages on the https://developer.mozilla.org/en-US/docs/Web/API/ShadowRoot/host page, but otherwise updated everything.